packages/surveys/src/components/questions/rating-question.tsx

```
@@ -111,179 +111,175 @@ export function RatingQuestion({
+            <legend className="fb-sr-only">Choices</legend>
+            <div className="fb-flex fb-w-full">
+              {Array.from({ length: question.range }, (_, i) => i + 1).map((number, i, a) => (
+                <span
+                  key={number}
+                  onMouseOver={() => {
+                    setHoveredNumber(number);
+                  }}
+                  onMouseLeave={() => {
+                    setHoveredNumber(0);
+                  }}
+                  className="fb-bg-survey-bg fb-flex-1 fb-text-center fb-text-sm">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)